### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_NewJobSteps_ColorContrast:Disable button is not clearly visible in steps tab after applying solarized light theme.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#6001]
**Bug Description:**
[Check out Keros!](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx) - Identify accessibility bugs before check-in and make bug fixing faster and easier.

GitHubTags: #A11yMAS; #A11yTCS; #MAS1.4.3; #A11y_AzureDataStudio; #Win32; #DesktopApp

**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.17763
**Additional Details:**
Tool Name: ColorContrastAnalyser
MAS Reference: MAS1.4.3
**Repro Steps:**
1. Install the app Azure Data Studio.
2.Connect to Database-> Navigate to the SQL agent tab from Home page->Click on new job button->Select schedule tab.
3.Navigate to step tab.
4.Go to settings and select colour theme and apply solarized light theme.
5.Observe the disabled button on steps tab.
**Actual:**
Disabled button "Move step up, Move step down, edit step and delete step." in steps tab  is not clearly visible in solarized light theme.

**Expected:**
It should be visible applying any theme.
User Impact:
User with visual disabilities will not be able view the button. 

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:MAS1.4.3**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

Attachment for Reference:
![6001_ Accessibility A11y_AzureDataStudio_NewJobSteps_ColorContrast](https://user-images.githubusercontent.com/39204909/59827220-b7671480-9355-11e9-92dd-a8f6d0067149.png)

Does this issue occur when all extensions are disabled?: Yes
